sqlglot/parser.py

```diff
@@ -394,6 +394,7 @@ class Parser(metaclass=_Parser):
         TokenType.NULL,
         TokenType.NAME,
         TokenType.TDIGEST,
+        TokenType.DYNAMIC,
```

Comment:
I think we should only map `"DYNAMIC"` to `TokenType.DYNAMIC` in Clickhouse, so this entry isn't actually needed. The token will be included due to `ENUM_TYPE_TOKENS`.
```suggestion
```